packages/app/src/app/pages/NewDashboard/Content/routes/Templates/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { Element, Text } from '@codesandbox/components';
+import { useQuery } from '@apollo/react-hooks';
+import {
+  ListTemplatesQuery,
+  ListTemplatesQueryVariables,
+} from 'app/graphql/types';
+import { LIST_OWNED_TEMPLATES } from 'app/components/CreateNewSandbox/queries';
+import { SandboxCard } from '../../../Components/SandboxCard';
+
+export const Templates = () => {
+  const { loading, error, data } = useQuery<
+    ListTemplatesQuery,
+    ListTemplatesQueryVariables
+  >(LIST_OWNED_TEMPLATES, {
+    variables: { showAll: false },
+  });
+
+  if (error) {
+    return <Text>Error</Text>;
+  }
+
+  if (loading) {
+    return <Text>loading</Text>;
+  }
+
+  const templates = data && data.me && data.me.templates;
```

Comment:
Since we know for sure there's no error or loading state, we know `data.me.templates` is present

```suggestion
  const templates = data.me.templates;
```